src/Http/Controllers/CourseAPIController.php

```diff
@@ -356,4 +355,142 @@ public function category(int $category_id, Request $request)
         $courses = $this->courseServiceContract->searchInCategoryAndSubCategory($category);
         return $this->sendResponse($courses->toArray(), 'Course updated successfully');
     }
+
+    /**
```

Comment:
For more readability consider moving OpenAPI annotations to Interface implemented by this Controller